cmd/controlplane/api.go

```diff
@@ -156,6 +158,11 @@ func newSchemeForAPI() (*runtime.Scheme, error) {
 	if err := kubescheme.AddToScheme(scheme); err != nil {
 		return nil, errors.Wrap(err, "add Kubernetes api to scheme")
 	}
+	if types.MustParseBool(os.GetEnv("ROLLOUTS_INTEGRATION_ENABLED", "true")) {
```

Comment:
This is a spot where we might try to make an inference from the presence or absence of Rollouts CRDs.